SerilogAnalyzer/SerilogAnalyzer/DiagnosticAnalyzer.cs

```diff
@@ -150,7 +150,12 @@ private static void AnalyzeSymbol(SyntaxNodeAnalysisContext context)
                         }
                     }
                 }
-                else if (paramter.Name.StartsWith("propertyValue", StringComparison.Ordinal))
+                else if (parameter.Name.StartsWith("propertyValue", StringComparison.Ordinal))
+                {
+                    var location = argument.GetLocation().SourceSpan;
+                    arguments.Add(new SourceArgument { StartIndex = location.Start, Length = location.Length });
+                }
+                else if (parameter.IsParams && IsObjectArray(parameter))
```

Comment:
I'll fix this, if we don't go with the less strict approach (which would fix that anyhow).